setup/README.md

```diff
@@ -96,19 +97,32 @@ From the **Plugin Listing** tab, click on the `Get Plugins...` button near the u
 
 ![get_plugins](resources/get_plugins.png)
 
-In the **Get Plugins** window, select the `[FFXIV+others]Overlay Plugin` option and click on `Download and Enable`. This will add the latest ngld **OverlayPlugin** to ACT (the OverlayPlugin auto-updater may also run during this step).
+In the **Get Plugins** window, select the `[FFXIV+others]Overlay Plugin` option and click on `Download and Enable`. This will add the latest **OverlayPlugin** to ACT (the OverlayPlugin auto-updater may also run during this step). Note that the number displayed next to the plugin may differ from the one shown here.
 
 ![get_plugins_window](resources/get_plugins_window.png)
 
 The OverlayPlugin should now be setup. Click on the `X` to close the **Get Plugins** window.
 
 **At this point it is recommended to restart ACT before continuing on.**
 
+### Using OverlayPlugin to End Encounters
+
+It is recommended to use OverlayPlugin's in/out-of-combat detection to split encounters, rather than ACT's less accurate behavior.
+
+To do this, first set ACT's encounter split timeout (Options > Main Table/Encounters > General) to a large value, such as 60 seconds (or even higher, for duties with long cutscenes):
```

Comment:
Would it be worth recommending just unclicking this box entirely?

Reply:
Hmm, I don't know. I think it's still worth having some kind of value there for fresh patches when OP's memory sigs might not be updated yet.

Reply:
I sorta disagree.  Most of the time it'll break on wipes which come from ACT log lines, which is good enough for most use cases?  I've had it unchecked for a very long time with no issues.

Reply:
Mostly I also mention this because this is also Aho's advice in #general pins: https://cdn.discordapp.com/attachments/551474815727304706/921609700842299413/Step1.PNG

So, I figure it makes sense for the setup guide to match the pins? Or we change both of them?

Reply:
I guess I can just mention both ways of doing it

Reply:
You're saying that you disagree that the setup guide and the pins should have the same recommendation?

Reply:
Well, I didn't write the pin, so I can't vouch for the information.

There are a few situations where you're still "in combat" but it shouldn't be considered the same pull, e.g. Praetorium when people tag mobs and then hit the teleporter (unsure if it happens with the remade version). The ACT timer is more of a failsafe for those situations. 

Also, more generally, I think it's fine for a pin to be the short version, and then the guide can elaborate on it further.

Reply:
Belatedly, I don't think there's any vouching required here.  The pin says:

```
About : Handling ACT Encounters

Here's the recommendation : 
First, turn off that option in ACT Settings. 
>Then, you can either let OverlayPlugin handle it
>Or Triggernometry (Misc repo)
>Or you can make an in-game macro (but you'll have to press it everytime)
```
![image](https://user-images.githubusercontent.com/462317/229844695-806156ad-37eb-403c-81ab-b605f7fb2eb4.png)
(plus some other images about how to have trigg or overplayplugin handle it)

Belatedly, I remember that 60 seconds is not enough for some fight cutscenes, which is why I also think "don't click the checkbox" is a better recommendation.  I also continue to think that the pin in the discord and this guide should have the same advice to be less confusing to users.